.github/workflows/workflow.yml

```diff
@@ -56,6 +61,43 @@ jobs:
           rustup default nightly
           cd modules/rustbitcoin && make cargo && make
 
+      - name: Build - rust miniscript
+        timeout-minutes: 5
+        run: |
+          export CC=/usr/bin/clang
```

Comment:
Instead of repeating the export CC/CXX, we could set it "globally", e.g.:
```
      - name: Setup common environment
        run: |
          export CC=/usr/bin/clang
          export CXX=/usr/bin/clang++
          echo "CC=/usr/bin/clang" >> $GITHUB_ENV
          echo "CXX=/usr/bin/clang++" >> $GITHUB_ENV
```